snuba/web/rpc/v1/endpoint_get_traces.py

```diff
@@ -69,6 +76,16 @@
         AttributeKey.Type.TYPE_STRING,
     ),
 }
+# for every AttributeKey of TYPE_FLOAT a user may add during the backward compat period, this adds the TYPE_DOUBLE equivalent
```

Comment:
We're not letting users choose the type they want in this API. I would remove all the compatibility and just change the timestamp type to `TYPE_DOUBLE`.